src/classes/BDI_CustObjMappingGAUAllocation.cls

```diff
@@ -44,8 +44,8 @@ public with sharing class BDI_CustObjMappingGAUAllocation extends BDI_ObjectMapp
 
         Map<String, Schema.DescribeFieldResult> sourceFieldDescByFieldName = UTIL_Describe.getAllFieldsDescribe(SObjectType.DataImport__c.getName());
 
-        String PAYMENT_FIELDNAME = UTIL_Namespace.adjustNPSPNamespace('npsp__Payment__c');
-        String OPPORTUNITY_FIELDNAME = UTIL_Namespace.adjustNPSPNamespace('npsp__Opportunity__c');
+        String PAYMENT_FIELDNAME = UTIL_Namespace.alignSchemaNSWithEnvironment('npsp__Payment__c');
```

Comment:
How does this compare with using something like:

SObjectType.Allocation__c.fields.Payment__c.Name

It seems like if we can get the platform to give use the result we want that would be nice...?

Reply:
It looks to be the same, I can make the change there, and probably in a few more places.  I'm assuming that there would be no difference in performance since this info should be cached.

Reply:
@ptewson-sfdo the latest commit should remove these hardcoded names as well as a few others.